mathesar/api/serializers/databases.py

```diff
@@ -23,33 +23,10 @@ def get_supported_types_url(self, obj):
         if isinstance(obj, Database) and not self.partial:
             # Only get records if we are serializing an existing table
             request = self.context['request']
-            return request.build_absolute_uri(reverse('database-types', kwargs={'pk': obj.pk}))
+            return request.build_absolute_uri(reverse('connection-types', kwargs={'pk': obj.pk}))
```

Comment:
There is a minor bug here, the uri formed will always be namespaced under `ui/` as the `basename` is same for both the `db` & `ui` route. Not sure how significant it is for the frontend though.

Reply:
Hmmm.  I'll experiment with it a bit.

Reply:
@Anish9901 I poked around and found that the behavior's the same as before. I agree that this is a bit of a bug, but it's apparently harmless. Given that we're going to be completely overhauling the API structure in the near future, I want to avoid fixing it for now.